src/components/LoginButton.tsx

```diff
@@ -0,0 +1,73 @@
+import { Button, Popover, Spin } from "antd";
+import * as wallet from "../services/wallets/wallet";
+import React from "react";
+
+interface LoginButtonProps {
+  popoverVisible: boolean;
+  setPopoverVisible: any;
+  loginWalletOptions: wallet.WalletType;
+  loading: boolean;
+  onButtonClick: (wallet: wallet.WalletType) => void;
+}
+
+const LoginButton = ({
+  popoverVisible,
+  setPopoverVisible,
+  loginWalletOptions,
+  loading,
+  onButtonClick,
+}: LoginButtonProps): JSX.Element => {
+  const handleVisibleChange = (visible: boolean) => {
+    setPopoverVisible(visible);
+  };
+
+  const getHeaderLoginButton = (
+    <Popover
```

Comment:
Delete popover? UX is confusing to have multiple of the same thing.  lets simplify.